app/src/components/market/sections/market_create/steps/items/funding_and_fee_step.tsx

```diff
@@ -200,6 +203,9 @@ const FundingAndFeeStep: React.FC<Props> = (props: Props) => {
     values,
   } = props
   const { arbitrator, category, collateral, funding, loadedQuestionId, outcomes, question, resolution, spread } = values
+  const { markets } = useGraphMarketsFromQuestion(loadedQuestionId || '')
+  const [currentToken, setCurrentToken] = useState({ exists: false, token: '' })
+  const allMarkets = markets.map(item => item.collateralToken)
```

Comment:
@Mi-Lan , could you rename `allMarkets` to `marketTokens` or other?

Reply:
@liaojikunwork ok!